doc/src/stdlib/linalg.md

```diff
@@ -73,6 +73,10 @@ Base.LinAlg.diagm
 Base.LinAlg.scale!
 Base.LinAlg.rank
 Base.LinAlg.norm
+Base.LinAlg.norm(::AbstractVector, ::Real)
```

Comment:
As far as I can tell, it's the `function norm end` line to blame here -- then with `Base.LinAlg.norm`, Documenter only grabs the docstring attached to that particular declaration. Same behaviour in a simple example. I'm guessing because it's in this case an exact match.. but I am not at all familiar with the docsystem code (neither in Base nor in Documenter).

Not sure whether this should be considered to be a bug or not though. Without this there's no way to exactly match the `function foo end` docstring I believe, but on the other hand it conflicts with the "grab all docstrings" semantics.